docs/source/fixtures.md

```diff
@@ -0,0 +1,22 @@
+# Fixtures and Mocks of the `jupyterhub` package
+
+(jupyterhub-mockhub)=
+## [The MockHub Class](https://github.com/jupyterhub/jupyterhub/blob/e4f72c9eeb4cd308ff5cbcf21142b2cb0a0345e4/jupyterhub/tests/mocking.py#L220) 
+It is a subclass of `JupyterHub` that allows for easier testing of the `JupyterHub` class and other components that require a running instance of JupyterHub. It does this by providing [mock](https://docs.python.org/3/library/unittest.mock.html) implementations of particular functionality of a JupyterHub instance. It also allows for easier setup and cleanup of the testing environment.
```

Comment:
"implementations of particular functionality"
What does "particular functionality" mean?

Reply:
> "implementations of particular functionality" What does "particular functionality" mean?

Hey @manics,
By "particular functionality", I mean that it provides some specific features belonging to the JupyterHub class, sometimes modified for testing.
Is there a way you think I could word it better?

Reply:
@Sheila-nk, I believe writing the explanation you just gave 👍🏼 , together with an example of these pieces that are being simplified by mocking (like initializing the hub database, initializing the hub and stopping and cleaning up the resources used by this MockHub during a test) should be enough for a more clear picture of the things that are mocked. What do you think?

Reply:
Yes, I think that works. I will add that information to give a clearer picture.